src/ImageGallery.jsx

```diff
@@ -1389,16 +1390,13 @@ export default class ImageGallery extends React.Component {
                   </React.Fragment>
                 )
               }
-              <Swipeable
+              <SwipeWrapper
```

Comment:
We don't need a new component here. Can we define the handlers above and pass it in. 

Similar to the example on https://github.com/FormidableLabs/react-swipeable
```jsx
const handlers = useSwipeable({
  onSwiped: (eventData) => console.log("User Swiped!", eventData),
  ...config,
});
return <div {...handlers}> You can swipe here </div>;
```

Reply:
useSwipeable is a hook and can only be used in a functional component. 

Since the current implementation of ImageGallery is a class component, I don't think it is possible.